src/Turbo/src/EventSubscriber/WebDebugToolbarTurboDriveFixSubscriber.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo\EventSubscriber;
+
+use Symfony\Component\DependencyInjection\Attribute\When;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Allows the Web Debug Toolbar (WDT) when navigating via Turbo Drive when a strict Content-Security-Policy is set.
+ * This is done by reusing WDT nonces.
+ */
+#[When(env: 'dev')]
+class WebDebugToolbarTurboDriveFixSubscriber implements EventSubscriberInterface
+{
+    public function onKernelResponse(ResponseEvent $event): void
+    {
+        $request = $event->getRequest();
+        $routeName = $request->get('_route');
```

Comment:
You need to check 

        if (!$this->kernel->isDebug()) {
            return;
        }
        
(see: https://symfony.com/doc/current/profiler.html#updating-the-web-debug-toolbar-after-ajax-requests)